tests/test_jwt.py

```diff
@@ -59,7 +59,7 @@ def test_sign_and_verify(self):
     self._check_sign_and_verify('privatekey.%s' % self.format)
 
   def test_sign_and_verify_from_converted_pkcs12(self):
-    """Tests that following instructions to convert from PKCS12 to PEM works."""
```

Comment:
Couldn't find anything in a few minutes of searching. I'm pretty sure I've seen something before.

Closest I got was the `unittest` docs for making the description ignore the docstring:
https://docs.python.org/2/library/unittest.html#unittest.TestCase.shortDescription

Maybe @tseaver has a quick reference